board/safety/safety_ford.h

```diff
@@ -255,13 +255,18 @@ static int ford_tx_hook(CANPacket_t *to_send) {
     bool violation = (raw_curvature_rate != INACTIVE_CURVATURE_RATE) || (raw_path_angle != INACTIVE_PATH_ANGLE) || (raw_path_offset != INACTIVE_PATH_OFFSET);
 
     int desired_curvature = raw_curvature - INACTIVE_CURVATURE;  // /FORD_STEERING_LIMITS.angle_deg_to_can to get real curvature
-    if (controls_allowed) {
+    if (controls_allowed && steer_control_enabled) {
       if (vehicle_speed > FORD_CURVATURE_DELTA_LIMIT_SPEED) {
         violation |= angle_dist_to_meas_check(desired_curvature, &angle_meas,
                                               FORD_STEERING_LIMITS.max_angle_error, FORD_STEERING_LIMITS.max_steer);
       }
     }
 
+    // If steer control is not enabled, curvature must be 0
+    if (!steer_control_enabled && (desired_curvature != 0)) {
+      violation = true;
+    }
+
```

Comment:
Would be nice if we can just copy what commonangle safety does (always send it and assert it matches meas), but I don't think that can work here.

1. Curvature is limited to 0.02, which is much lower than what you can achieve,
2. and the system takes a while to start applying torque if desired curvature is non-zero when you set the apply bit

So we'll have to handle this case when we merge with common angle safety in https://github.com/commaai/panda/pull/1369